stdlib/@tests/test_cases/builtins/check_pow.py

```diff
@@ -17,7 +17,11 @@
 # assert_type(pow(2, 4, 0), NoReturn)
 
 assert_type(pow(2, 4), int)
-assert_type(2**4, int)
+# pyright infers a literal type here, but mypy does not. Unfortunately,
+# there is no way to ignore an error only for mypy, so we can't check
+# pyright's handling (https://github.com/python/mypy/issues/12358).
```

Comment:
Pedantic correction: (um aktually 🤓) that is not quite correct, that's what `"enableTypeIgnoreComments": false,` is for in pyrightconfigs

It's just that `pyrightconfig.testcases.json` sets it explicitely to `true` "for simplicity"

So we can't have mypy specific disables *and* have both respect `type: ignore`

There's almost a solution with control comments: `# pyright: enableTypeIgnoreComments=false` but pyright says `"enableTypeIgnoreComments" is an unknown diagnostic rule for pyright comment` >.<